third_party/terraform/tests/resource_storage_bucket_acl_test.go

```diff
@@ -5,7 +5,6 @@ import (
 	"os"
 	"testing"
 
-	"github.com/hashicorp/terraform/helper/acctest"
```

Comment:
These changes are unintentional, right?

Reply:
I took the liberty to move this function into the file below. I can drop that change if your rather keep that untouched. (But it made a bit more sense that it was located there and didn't specifically add the `-acl` part)